packages/flutter/lib/src/cupertino/switch.dart

```diff
@@ -467,6 +491,14 @@ const double _kTrackInnerEnd = _kTrackWidth - _kTrackInnerStart;
 const double _kTrackInnerLength = _kTrackInnerEnd - _kTrackInnerStart;
 const double _kSwitchWidth = 59.0;
 const double _kSwitchHeight = 39.0;
+// Label sizes and padding taken from xcode inspector.
+// See https://github.com/flutter/flutter/issues/4830#issuecomment-528495360
+const double _kOnLabelWidth = 1.0;
```

Comment:
The measurements here are a bit different from https://github.com/flutter/flutter/pull/39993 (e.g. the stroke width being 1px instead of 2px in the previous PR)?

Reply:
I explained why these values are half the value shown in the inspector in [this thread](https://github.com/flutter/flutter/pull/127776#discussion_r1231306321).

The width in #39993 is actually 1.25px, I don't know why! See: https://github.com/flutter/flutter/pull/39993/files#diff-c8dc74e1119fcfc41cb08a151f40285dc4cde603a5da5d66b53b1dd6b7ad5529R531

Reply:
Oh are the values in #39993 in device pixels instead of logic pixels (UIKit points)? I thought the values were taken from the view hierarchy inspector so they should be in points.

Reply:
is 1.0px ok for the width then?